_posts/2020-05-07-report-deep-sea-robotics.md

```diff
@@ -0,0 +1,182 @@
+
+![figure1](/assets/2020-05-07-report-deep-sea-robotics/1theUnderwaterWorldZoomedOut.png)
+![figure2](/assets/2020-05-07-report-deep-sea-robotics/2CodeofDeepSeaEnvironment.png)
+
+The depth I defined was 2000 meters. The final dimensions are 8000x8000x2000 meters. The underwater world has pressure and density of water. One way that I tested the buoyancy of the simulator was by verifying denser objects sunk (Figure 3). 
+
+![figure3](/assets/2020-05-07-report-deep-sea-robotics/3CubeSinkinginSeaEnvironment.png)
+
+Afterwards, I combined the lake environment (Figure 4) with the empty underwater world environment to build a deep sea environment with varied topography at the sea floor. 
+
+![figure4](/assets/2020-05-07-report-deep-sea-robotics/4seaFloorofDeepSeaEnvironment.png)
+
+
+## Robot
+### Initial Research
+Prior to simulating my aquatic robot and making design choices, I studied how different materials and shapes interact with deep sea pressure, existing deep sea robots, and animal adaptations to the deep sea. 
+
+#### Shapes and Materials
+One of the biggest challenges to building a robot underwater, as opposed to land is the difference in pressure. Pressure is defined as the force per unit area. Unlike gas, water does not exert equal pressure on all sides. More pressure is exerted on the bottom of the object than the top. Combined with the descent of an object in water, pressure is constantly changing. For each 10 meters of depth, the pressure increases by one atmosphere.  
+	To counteract the forces of pressure on the material, wall tension occurs in the casing of the material.  Wall tension is proportional to the radius and the pressure exerted. The most common shapes used for pressure vessels are spheres and cylinders. The cylinder’s wall tension or hoop stress is calculated as: 
+	
+![figure5](/assets/2020-05-07-report-deep-sea-robotics/5.png)
+
+Where Ap= projected area, dy = incremental  length, t = wall  thickness, r =  inner  radius,  p  =  gauge  pressure, and  σh is the hoop stress/wall tension.
+The sphere’s calculated wall tension: 
+
+![figure6](/assets/2020-05-07-report-deep-sea-robotics/6.png)
+
+Where ro = inner radius and σa is the axial stress/wall tension. Both wall tensions are calculated assuming that the object is at equilibrium; the sum of the forces is 0. The resulting wall tension of a sphere, with a given radius r is two times smaller than that of the cylinder (Bednar, 1981). 
+
+![figure7](/assets/2020-05-07-report-deep-sea-robotics/7.png)
+	Thus, the sphere is the most efficient shape to withstand pressure because of its equal distribution on all sides and its minimal surface area.  Fundamentally, this makes the sphere a better candidate for pressure vessels. However, due to the high cost of manufacturing a sphere vessel, a cylinder is the more common pressure vessel in current existing studies (Geelhoed, 2016).
```

Comment:
```suggestion

Thus, the sphere is the most efficient shape to withstand pressure because of its equal distribution on all sides and its minimal surface area.  This makes the sphere a better candidate for pressure vessels. However, due to the high cost of manufacturing a sphere vessel, a cylinder is the more common pressure vessel in current existing studies (Geelhoed, 2016).
```